build/AzurePipelinesTemplates/WindowsAppSDK-BuildInstaller-Steps.yml

```diff
@@ -27,12 +27,14 @@ steps:
   # This step downloads the WindowsAppSDK NuGet package from last successful build of the Aggregator's Nightly build
```

Comment:
Nit: comment needs update to include failed builds?

Reply:
oh good call out